### DISCUSSION TITLE
Landslide Susceptibility Mapping using Machine Learning (Project 228)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#62] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #228](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Landslide%20Susceptibility%20Mapping%20using%20Machine%20Learning)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Emma2020-star** [comment]
Hell, how do  l access the training data

**Reply by mw-pc** [reply]
Hello, section #2 of reference #1 points to the following https://github.com/jeffreyevans/GradientMetrics as a means in which the authors obtained the necessary data e.g., Distance from roads, distance from geological faults, distance from water streams, land use type, geology type, etc.

**Reply by ankur18439** [reply]
Hi @Emma2020-star! I think we'll have to generate the input data ourselves. Some organizations provide landslide inventories for data labeling (I remember NASA has a publicly available landslide inventory that you can use). The other datasets will have to be processed using digital elevation models and satellite imagery.

**Comment by acetarik** [comment]
Hey! How do I access the dataset for training my nn?

**Reply by Emma2020-star** [reply]
hello, you can navigate to the various portals that track landslide occurrences, they have a page for their datasets.

**Reply by acetarik** [reply]
Thanks! It would be helpful if you could provide me with a link, it'll let me know if i am on the right page or not :3

**Reply by mw-pc** [reply]
Hello, consider search IEEE Explore.  Here's an [example search result.](https://ieeexplore.ieee.org/search/searchresult.jsp?newsearch=true&queryText=landslide%20risk%20prediction%20dataset)

**Reply by ankur18439** [reply]
Hey @acetarik! It's not a one-stop solution - you'll have to compile it from at least 2-3 data sources. You can digitize your labeling data (the landslide locations) in GIS but it is too much of a hassle sadly.
